src/api.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+const API_URL = 'https://api.datavortex.nl/deckbuildermtg';
+
+const headers = {
+    'Content-Type': 'application/json',
+    'X-Api-Key': 'deckbuildermtg:6rk6IsJiCtWHtF7OgI2B',
```

Comment:
Dit soort dingen wil je in een .env bestand bewaren, zodat het niet in je source-code staat. Dan blijft het geheim. Weet niet of je hierover al geleerd hebt? :)